nsxt/resource_nsxt_compute_manager.go

```diff
@@ -449,7 +449,7 @@ func setCredentialValuesInSchema(d *schema.ResourceData, credential *data.Struct
 		return errors.New("no valid credential found")
 	}
 
-	d.Set("credential", elem)
+	d.Set("credential", []interface{}{elem})
```

Comment:
was this wrong before and we did not realize only because elem was an empty map?

Reply:
Indeed. And the list was empty as the constants which were used in the switch case were wrong as well. So values were taken from the state.